niri-ipc/src/lib.rs

```diff
@@ -425,6 +418,19 @@ pub struct ConfiguredPosition {
     pub y: i32,
 }
 
+/// Output VRR to set.
+#[derive(Serialize, Deserialize, Debug, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "clap", derive(clap::ValueEnum))]
+#[cfg_attr(feature = "json-schema", derive(schemars::JsonSchema))]
+pub enum VrrToSet {
+    /// Always OFF.
+    AlwaysOff,
+    /// Always ON.
+    AlwaysOn,
+    /// Only ON when a surface matching a VRR window rule is visible on the output.
+    OnDemand,
+}
```

Comment:
I wonder if it's better to match the config here, i.e. instead of `niri msg output vrr always-on/always-off/on-demand` have it be:
- `niri msg output vrr off`
- `niri msg output vrr on`
- `niri msg output vrr on --on-demand`

Reply:
I am not sure if this is possible under clap.